components/script/dom/bindings/typedarrays.rs

```diff
@@ -31,6 +37,13 @@ where
     T: TypedArrayElement + TypedArrayElementCreator,
     T::Element: Clone + Copy,
 {
+    pub fn new(internal: *mut JSObject) -> HeapTypedArray<T> {
```

Comment:
Just as a note: in the other PR we should change this to `new_initialized: internal: Option<*mut JSObject>`.